### PR TITLE
Fix examples/with-csr

### PR DESCRIPTION
**Pull request by ahuigo**
1. fix bad assert for content-type and body in unittest
  2. fix bad importMap path in development state

### CODE REVIEW

File: examples/with-csr/server.test.ts [hunk 1]
```diff
@@ -12,10 +12,10 @@ Deno.test("it works", async (t) => {
 
     assertEquals(response.status, 200);
     assertEquals(
-      response.headers.get("content-type"),
+      (response.headers.get("content-type") || "").toLowerCase(),
       "text/html; charset=utf-8",
```
Comment (ahuigo) [inline examples/with-csr/server.test.ts:16]: Hono uses "text/html; charset=UTF-8" instead.

File: examples/with-csr/server.tsx [hunk 2]
```diff
@@ -22,7 +23,7 @@ server.get("*", (context) => {
         <link rel="stylesheet" href="/style.css">
         <script async src="https://ga.jspm.io/npm:es-module-shims@1.6.2/dist/es-module-shims.js" crossorigin="anonymous"></script>
         <script type="importmap">
-            ${Deno.readTextFileSync("./importMap.json")}
+            ${Deno.readTextFileSync(importMapPath)}
```
Comment (ahuigo) [inline examples/with-csr/server.tsx:26]: With `ULTRA_MODE=development`, this code should use `importMap.dev.json` here.

**Review by deckchairlabs — APPROVED** [review]
Thanks! Could you rebase this PR?

**Comment by deckchairlabs** [comment]
Test failures are unrelated, I can work on fixing those tomorrow.

**Comment by ahuigo** [comment]
Since this bug has been fixed in https://github.com/exhibitionist-digital/ultra/pull/238, this PR should be closed.

**Pull request closed without merging**